es6/jsxcad-api-v1-png.js

```diff
@@ -129,10 +13,7 @@ const readPng = async (path) => {
   return raster;
 };
 
-const readPngAsContours = async (
-  path,
-  { by = 10, tolerance = 5, to = 256 } = {}
-) => {
+const readPngAsContours = async (path, bands = [128, 256]) => {
```

Comment:
How does this work? It pulls in all the pixels as points?

Reply:
The pixels are converted to contour lines based on the grayscale value -- like on a map.

They come in as segments.